doc/getting_started/setup_fpga.md

```diff
@@ -136,6 +136,25 @@ When properly connected, `dmesg` should identify the board, not show any errors,
 They should be named `'/dev/ttyACM*'`, e.g. `/dev/ttyACM1`.
 To ensure that you have sufficient access permissions, set up the udev rules as explained in the [Vivado installation instructions]({{< relref "install_vivado" >}}).
 
+You will then need to run this command to configure the board. You only need to run it once.
+```console
+bazel run //sw/host/opentitantool -- --interface=cw310 fpga set-pll
+```
+
+Check that it's working by [running the demo]({{< relref "#hello-world-demo" >}}) or a test, such as the `uart_smoketest` below.
+```console
+cd $REPO_TOP
+bazel test --test_output=streamed //sw/device/tests:uart_smoketest_fpga_cw310_test_rom
+```
+
+### Troubleshooting
+
+If the tests/demo aren't working on the FPGA (especially if you get an error like `SFDP header contains incorrect signature`) then try adding `--rcfile=` to the `set-pll` command:
+```console
+bazel run //sw/host/opentitantool -- --rcfile= --interface=cw310 fpga set-pll
+```
+It's also worth pressing the `USR_RST` button on the FPGA if you face continued errors.
```

Comment:
Do you mean USR_RESET or USB_RST? I've found USB_RST useful in the past when setting up the board and meant to mention it in the documentation somewhere when I set up my FPGA.

Reply:
I'll just add both here. `USR_RST` is the one I ended up pressing, but no idea if it did anything and I've also found `USB_RST` helpful in the past.

Reply:
Just a further nit here: Silkscreen labels can be hard to correctly read and the USB_RST label is fairly small, On my CW310, the other label reads "USR_RESET" and being careful about the labels gives people a better chance of reading it correctly.

Reply:
I can confirm: "USB_RST" is mentioned elsewhere

Reply:
Where it's associated with "SW5"